projects/aca-content/src/lib/components/header/header.component.html

```diff
@@ -1,19 +0,0 @@
-<adf-layout-header
```

Comment:
We should not remove header html tags here, if we do not need this component, we should remove the whole component, or remove usage of this component. 

Now we are going to have `header.component` without any UI to display